### DISCUSSION TITLE
How to create a rule based on "Inbound"?

### DISCUSSION THREAD

**Discussion by turtle-ca-chan** [vernesong/OpenClash#4135] [Q&A] [answered]
Hi! Can you please advise if it is possible to create a rule based on "Inbound"? I have tried using `'INBOUND,Socks5'` but this gives the error `"parse rule error:[unsupported rule type Inbound]"`
I need to set such rule to enable full traffic proxying for any application which uses OpenClash's proxy server on `192.168.2.1:7891` (this is `[My router IP]:[Socks5 Port]`)
I don't need to enable proxies globally and I still want to use customized proxy rules for most sites. But I do need to be able to proxy all application traffic if I setup OpenClash's proxy server `192.168.2.1:7891` for it.
It seems to me that such implementation should be possible, **because OpenClash distinguishes between Inbound types:**
![screen](https://github.com/user-attachments/assets/0751329d-e60a-4552-a319-be439ec97bf5)

**Comment by morytyann** [comment, marked as answer]
https://wiki.metacubex.one/config/rules/#in-type

**Reply by turtle-ca-chan** [reply]
Thank you!
